docs/2.deploy/20.providers/cloudflare.md

```diff
@@ -85,16 +85,14 @@ The following shows a typical `wrangler.toml` file for a Nitro application:
 name = "playground"
 main = "./.output/server/index.mjs"
 workers_dev = true
-compatibility_date = "2023-12-01"
+compatibility_date = "2024-11-01"
```

Comment:
I think we also need to document `compatibilityDate` has to be set in nitro/nuxt config to enable new preset (we could also use two tabs perhaps for now for legacy and new one)